src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -1151,6 +1151,11 @@ public ErrorDiagnostic UnknownModuleReferenceScheme(string badScheme, ImmutableA
                 TextSpan,
                 "BCP193",
                 $"The specified OCI artifact reference \"{badRef}\" is not valid. Specify a reference in the format of \"oci:<artifact uri>:<tag>\".");
+
+            public ErrorDiagnostic InvalidTemplateSpecReference(string invalidReference) => new(
+                TextSpan,
+                "BCP194",
+                $"The specified template spec reference \"{invalidReference}\" is not valid. Specify a reference in the format of \"ts:<resourceGroupName>/<templateSpecName>:<tag>\" or \"ts:<subscriptionId>/<resourceGroupName>/<templateSpecName>:<tag>\".");
```

Comment:
>ts [](http://example.com/codeflow?start=132&length=2)

can you point this and the other occurrence to `ModuleReferenceSchemes.TemplateSpecs`? I'm doing the same in my PR for OCI in the error above.

Reply:
Makes sense. Will do.

Reply:
Btw, I'll improve the error message in a separate PR.